### PR TITLE
Add add-title service

### PR DESCRIPTION
**Pull request by orballo**
fixes https://github.com/frontity/demos-micros/issues/6

### CODE REVIEW

**Comment by orballo** [comment]
@luisherranz ready for review 😃

File: services/add-title/src/index.js [hunk 1]
```diff
@@ -1,4 +1,93 @@
-const { send } = require('micro');
+const { parse } = require('url');
+const { send, createError } = require('micro');
 const cors = require('micro-cors')();
+const axios = require('axios');
 
-module.exports = cors((req, res) => send(res, 200, 'In and out, twenty minutes adventure!'));
+module.exports = cors(async (req, res) => {
+  try {
+    const [initialUrl] = req.url.replace('/', '').match(/http.+$/) || [''];
```
Comment (luisherranz) [inline services/add-title/src/index.js:8]: I like this one 👆 never thought of solving `null` regexps like that.

**Review by luisherranz — APPROVED** [review]
Great code, clear and well organized. Good work 👍

**Pull request merged**